### PR TITLE
terraform fmt, and fixing tf 0.12 warnings

### PR DESCRIPTION
**Pull request by WillBeebe**
Thanks for making this module, it's really helpful! I ran `terraform fmt` and fixed all the 0.12 warnings that pop up when you run an apply.

Cheers!

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -53,8 +53,9 @@ provider "kubernetes" {
 
 // Services
 module "project_services" {
-  source  = "terraform-google-modules/project-factory/google//modules/project_services"
-  version = "~> 8.0"
+  # source  = "terraform-google-modules/project-factory/google//modules/project_services"
+  source = "github.com/WillBeebe/terraform-google-project-factory//modules/project_services"
```
Comment (bharathkkb) [inline main.tf:57]: Is there a reason why this pointing to a fork?

File: main.tf [hunk 2]
```diff
@@ -188,72 +189,73 @@ resource "google_redis_instance" "gitlab" {
...
 resource "google_storage_bucket" "gitlab-runner-cache" {
   name     = "${var.project_id}-runner-cache"
-  location = "${var.region}"
+  location = var.region
 }
 // GKE Cluster
 module "gke" {
-  source = "terraform-google-modules/kubernetes-engine/google"
-  version = "~> 9.0"
+  source            = "github.com/WillBeebe/terraform-google-kubernetes-engine"
```
Comment (bharathkkb) [inline main.tf:231]: pointing to fork

**Comment by WillBeebe** [comment]
Oops, sorry Bharath, I'll clean up this PR.

**Comment by bradkwadsworth** [comment]
Any idea on when this might get merged? I feel like this may be required in order to also work on #43.

**Pull request merged**